packages/zarith/zarith.1.3/files/xen_linkopts.patch

```diff
@@ -2,10 +2,7 @@ diff --git a/META b/META
 index c082c65..14394d1 100644
 --- a/META
 +++ b/META
-@@ -1,5 +1,6 @@
- description = "Arbitrary precision integers"
--requires = ""
-+requires = "gmp"
+@@ -3,3 +3,4 @@
```

Comment:
Or maybe requiring an optionnally existing ocamlfind package works... I'll check that tomorrow on my xen box.